opm/parser/eclipse/EclipseState/Tables/VFPProdTable.cpp

```diff
@@ -385,6 +391,14 @@ void VFPProdTable::check() {
     }
 
     if (num_decreasing > 0) {
+        m_messages.warning("VFP table for production wells has BHP versus THP not " 
+                           + std::string("monotonically increasing.\nThis may cause convergence ")
```

Comment:
It is amazing how poor old-style C++ string handling is (and therefore also string literals)...

A better way to work around this (cannot do "foo" + "bar") problem is to concatenate the strings:
```
    std::string s("This is\n"
                  "one long"
                  " string.");
```